### ISSUE TITLE
More precise error message on failing statements

### ISSUE THREAD

**Issue by MKlimentowicz** [capacitor-community/sqlite#82]
Is it possible to have a more precise error message when a statement fails?

I have a Problem with this statement: 
`SELECT * FROM myTable WHERE entityId = '710870e6-d70e-498c-8ad6-756ff361e75f';`

I then get such return:
`{ "values":[],"message":"Query command failed" }`

Is it possible to get an error message that comes directly from SQL. For example, an error code? (https://www.sqlite.org/rescode.html)

P.S. I don´t really know why the statement is failing...

**Comment by jepiqueau** [comment]
@MKlimentowicz First thanks for using the plugin, Which release of the plugin are you using?. the UUID has been introduced in release 2.9.8. The error management has been enhanced in the 3.0.0-beta of the plugin through the use of try.. catch in order to bring back the SQLite message. Can you share the myTable schema and few inserts so i can try to replicate

**Comment by MKlimentowicz** [comment]
@jepiqueau Thanks for your fast answer! 

I use this version: `"@capacitor-community/sqlite": "^2.9.9"`


Here is the creation script for my schema:
```
CREATE TABLE IF NOT EXISTS drawings (
    id TEXT PRIMARY KEY NOT NULL,
    congregationId TEXT,
    prefix TEXT,
    creationTime TEXT NOT NULL,
    lastUpdated TEXT,
    featureCollection TEXT NOT NULL,
    printConfiguration TEXT
  );
  PRAGMA drawings_version = 1;
```

It has no entries I just expect an empty result but I get this (it is my own console.log): 
```
[MobileDatabase.load(drawings)]: statement=>SELECT * FROM drawings WHERE congregationId = '494f7713-4dfe-4a92-b1a9-75aadcd71022';< | results=>{"values":[],"message":"Query command failed"}```

**Comment by jepiqueau** [comment]
@MKlimentowicz  I am looking at it for the next release. In electron it is returning results=>{"values":[]}

**Comment by MKlimentowicz** [comment]
@jepiqueau Thank You! I have found that it is not an error at all. After I added a few records, there was no error. **But it would still be very useful to get more precise error messages**. Especially as I now have a problem adding several records at once and only get this information: `{"changes":{"changes":-1}}`

**Comment by jepiqueau** [comment]
@MKlimentowicz fix in 2.9.10

**Comment by MKlimentowicz** [comment]
@jepiqueau 
I get an error when I create my schema. I use `2.9.12` Version of your plugin. 
```
CREATE TABLE IF NOT EXISTS drawings (
  id TEXT PRIMARY KEY NOT NULL,
  congregationId TEXT,
  prefix TEXT,
  creationTime TEXT NOT NULL,
  lastUpdated TEXT,
  featureCollection TEXT NOT NULL,
  printConfiguration TEXT
);
PRAGMA drawings_version = 1;
```

This statement creates nothing and ends with an error. But the message I get is just that:

```
V/com.getcapacitor.community.database.sqlite.SQLite.Database: ExecuteSet InitChanges 0
V/com.getcapacitor.community.database.sqlite.SQLite.Database: Error executeSet: Index: 0, Size: 0
E/Capacitor/Console: File: http://localhost/main.js - Line 8177 - Msg: [MobileDatabase.executeSet]: changes=>|{"changes":-1}|<
E/Capacitor/Console: File: http://localhost/main.js - Line 8177 - Msg: [MobileDatabase.executeSet]: message="undefined"
```

When I use the 'execute' method, the schemas are created. But with 'executeSet' it does not work. This is how you can reproduce the error:
```
const statements = [
  {statement: "statement from above" , values: []}
];
const result = await database.executeSet(statements);
```

From my point of view, the error messages have not changed. Can you please investigate and improve this?

**Comment by jepiqueau** [comment]
@MKlimentowicz Hi, the schema creation must be done through the `execute` command and not through the `executeSet` command which is made for inputting a set of data into the database.
The error message system  has been completely refactor on the 3.0.0-beta releases. This refactoring process will not be implemented in the 2.9.x releases. So move to the 3.0.0-beta which will become the official latest release as soon as  the Ionic Capacitor team release the @capacitor/core release. Hope this clarify

**Comment by jepiqueau** [comment]
@MKlimentowicz Hi, i will close that issue to morrow if no news
